tests/rules/data/cp_verify_alarm/cp_verify_bias_stats.yaml

```diff
@@ -0,0 +1 @@
CR_NOISE', 'RXX_S00 C04 NOISE', 'RXX_S00 C04 PROCESSING', 'RXX_S00 C05 NOISE', 'RXX_S00 C05 PROCESSING', 'RXX_S00 C06 NOISE', 'RXX_S00 C06 PROCESSING', 'RXX_S00 C07 CR_NOISE', 'RXX_S00 C07 NOISE', 'RXX_S00 C07 PROCESSING', 'RXX_S00 C10 CR_NOISE', 'RXX_S00 C10 NOISE', 'RXX_S00 C10 PROCESSING', 'RXX_S00 C11 CR_NOISE', 'RXX_S00 C11 NOISE', 'RXX_S00 C11 PROCESSING', 'RXX_S00 C12 CR_NOISE', 'RXX_S00 C12 NOISE', 'RXX_S00 C12 PROCESSING', 'RXX_S00 C13 CR_NOISE', 'RXX_S00 C13 NOISE', 'RXX_S00 C13 PROCESSING', 'RXX_S00 C14 CR_NOISE', 'RXX_S00 C14 NOISE', 'RXX_S00 C14 PROCESSING', 'RXX_S00 C15 CR_NOISE', 'RXX_S00 C15 NOISE', 'RXX_S00 C15 PROCESSING', 'RXX_S00 C16 CR_NOISE', 'RXX_S00 C16 NOISE', 'RXX_S00 C16 PROCESSING', 'RXX_S00 C17 CR_NOISE', 'RXX_S00 C17 NOISE', 'RXX_S00 C17 PROCESSING']}, 2022110400019: {'FAILURES': ['RXX_S00 C00 NOISE', 'RXX_S00 C00 PROCESSING', 'RXX_S00 C01 CR_NOISE', 'RXX_S00 C01 NOISE', 'RXX_S00 C01 PROCESSING', 'RXX_S00 C02 CR_NOISE', 'RXX_S00 C02 NOISE', 'RXX_S00 C02 PROCESSING', 'RXX_S00 C03 CR_NOISE', 'RXX_S00 C03 NOISE', 'RXX_S00 C03 PROCESSING', 'RXX_S00 C04 CR_NOISE', 'RXX_S00 C04 NOISE', 'RXX_S00 C04 PROCESSING', 'RXX_S00 C05 NOISE', 'RXX_S00 C05 PROCESSING', 'RXX_S00 C06 NOISE', 'RXX_S00 C06 PROCESSING', 'RXX_S00 C07 CR_NOISE', 'RXX_S00 C07 NOISE', 'RXX_S00 C07 PROCESSING', 'RXX_S00 C10 CR_NOISE', 'RXX_S00 C10 NOISE', 'RXX_S00 C10 PROCESSING', 'RXX_S00 C11 CR_NOISE', 'RXX_S00 C11 NOISE', 'RXX_S00 C11 PROCESSING', 'RXX_S00 C12 CR_NOISE', 'RXX_S00 C12 NOISE', 'RXX_S00 C12 PROCESSING', 'RXX_S00 C13 CR_NOISE', 'RXX_S00 C13 NOISE', 'RXX_S00 C13 PROCESSING', 'RXX_S00 C14 CR_NOISE', 'RXX_S00 C14 NOISE', 'RXX_S00 C14 PROCESSING', 'RXX_S00 C15 CR_NOISE', 'RXX_S00 C15 NOISE', 'RXX_S00 C15 PROCESSING', 'RXX_S00 C16 CR_NOISE', 'RXX_S00 C16 NOISE', 'RXX_S00 C16 PROCESSING', 'RXX_S00 C17 CR_NOISE', 'RXX_S00 C17 NOISE', 'RXX_S00 C17 PROCESSING']}, 2022110400020: {'FAILURES': ['RXX_S00 C00 NOISE', 'RXX_S00 C00 PROCESSING', 'RXX_S00 C01 CR_NOISE', 'RXX_S00 C01 NOISE', 'RXX_S00 C01 PROCESSING', 'RXX_S00 C02 CR_NOISE', 'RXX_S00 C02 NOISE', 'RXX_S00 C02 PROCESSING', 'RXX_S00 C03 CR_NOISE', 'RXX_S00 C03 NOISE', 'RXX_S00 C03 PROCESSING', 'RXX_S00 C04 CR_NOISE', 'RXX_S00 C04 NOISE', 'RXX_S00 C04 PROCESSING', 'RXX_S00 C05 NOISE', 'RXX_S00 C05 PROCESSING', 'RXX_S00 C06 NOISE', 'RXX_S00 C06 PROCESSING', 'RXX_S00 C07 CR_NOISE', 'RXX_S00 C07 NOISE', 'RXX_S00 C07 PROCESSING', 'RXX_S00 C10 CR_NOISE', 'RXX_S00 C10 NOISE', 'RXX_S00 C10 PROCESSING', 'RXX_S00 C11 CR_NOISE', 'RXX_S00 C11 NOISE', 'RXX_S00 C11 PROCESSING', 'RXX_S00 C12 CR_NOISE', 'RXX_S00 C12 NOISE', 'RXX_S00 C12 PROCESSING', 'RXX_S00 C13 CR_NOISE', 'RXX_S00 C13 NOISE', 'RXX_S00 C13 PROCESSING', 'RXX_S00 C14 CR_NOISE', 'RXX_S00 C14 NOISE', 'RXX_S00 C14 PROCESSING', 'RXX_S00 C15 CR_NOISE', 'RXX_S00 C15 NOISE', 'RXX_S00 C15 PROCESSING', 'RXX_S00 C16 CR_NOISE', 'RXX_S00 C16 NOISE', 'RXX_S00 C16 PROCESSING', 'RXX_S00 C17 CR_NOISE', 'RXX_S00 C17 NOISE', 'RXX_S00 C17 PROCESSING']}, 'SUCCESS': False}
```

Comment:
Please pretty-print both of your yaml data files.

Reply:
I'd be happy to do so, but I can't find anything specific online to do this (i.e., to make a yaml file "pretty"). Did you have a package in mind or a particular standard/style that I should be following?

Reply:
Done :)

Reply:
Great! For the record, whenever I need this I search for "Yaml formatter" and pick one of the early hits.

This sort of thing is also useful for working with json schemas. Put the yaml through a yaml-to-json converter, then past that into a json schema validator (each of which is easily found with a web search). This shows any errors in the schema.